packages/gatsby/cache-dir/error-overlay-handler.js

```diff
@@ -1,18 +1,32 @@
-import * as ErrorOverlay from "react-error-overlay"
+import * as ReactRefreshErrorOverlay from "@pmmmwh/react-refresh-webpack-plugin/src/overlay"
+import * as ReactErrorOverlay from "react-error-overlay"
 
-// Report runtime errors
-ErrorOverlay.startReportingRuntimeErrors({
-  onError: () => {},
-  filename: `/commons.js`,
-})
-ErrorOverlay.setEditorHandler(errorLocation =>
-  window.fetch(
-    `/__open-stack-frame-in-editor?fileName=` +
-      window.encodeURIComponent(errorLocation.fileName) +
-      `&lineNumber=` +
-      window.encodeURIComponent(errorLocation.lineNumber || 1)
+const ErrorOverlay = {
+  showCompileError:
+    process.env.HOT_LOADER !== `fast-refresh`
+      ? ReactErrorOverlay.reportBuildError
+      : ReactRefreshErrorOverlay.showCompileError,
+  clearCompileError:
+    process.env.HOT_LOADER !== `fast-refresh`
+      ? ReactErrorOverlay.dismissBuildError
+      : ReactRefreshErrorOverlay.clearCompileError,
+}
+
+if (process.env.HOT_LOADER !== `fast-refresh`) {
+  // Report runtime errors
+  ReactErrorOverlay.startReportingRuntimeErrors({
+    onError: () => {},
+    filename: `/commons.js`,
+  })
+  ReactErrorOverlay.setEditorHandler(errorLocation =>
+    window.fetch(
+      `/__open-stack-frame-in-editor?fileName=` +
+        window.encodeURIComponent(errorLocation.fileName) +
+        `&lineNumber=` +
+        window.encodeURIComponent(errorLocation.lineNumber || 1)
+    )
   )
```

Comment:
@wardpeet  or @pieh any ideas on what these functions are doing? I can't tell if there is a need, or obvious way to port these to the new overlay.

Reply:
on _some_ errors you can click on call stack frames in browser and it will focus your code editor and jump to proper location

Reply:
Example:
![Kapture 2020-02-18 at 17 39 31](https://user-images.githubusercontent.com/419821/74756985-b1735b80-5275-11ea-9b11-be383fe6c1d9.gif)

Reply:
Looks like this functionality isn't supported in the new version. But there is exploration of supporting a custom overlay in the plugin. If that is implemented, we might be able to roll back to the react-overlay-error. But either way it's not a blocker for now.